src/entities/tokens.ts

```diff
@@ -110,3 +110,31 @@ export interface RainbowToken extends Asset {
 export interface IndexToken extends Asset {
   amount: string;
 }
+
+export interface ExchangeAsset {
```

Comment:
This is almost idential to the new type in my Swap PR https://github.com/rainbow-me/rainbow/pull/4014/files#diff-96cabbbbbd3a3fafc465e3b37f0fb18a4665266f6474670d0542c78e1df8acdeR114 but this one adds a missing property `network`, ive gone and added this prop to the other PR's interface so they should merge nicely